scalafmt-tests/src/test/resources/test/ContinuationIndentCaseSite5.case

```diff
@@ -0,0 +1,164 @@
+continuationIndent.caseSite = 5
+<<< Long if
+case tok if tok.left.name.startsWith("xml") && tok.right.name.startsWith("xml") => List( NoSplit0 )
```

Comment:
i just noticed that these are all bare `case` statements (commented for another file), it's better to use a valid language construct.

Reply:
Updated, please check again.